components/layout/MainHero.module.css

```diff
@@ -18,9 +18,11 @@
     text-align: center;
     margin: 0 auto;
     font-size:2vw;
+    padding-bottom: 2.5rem;
 }
 
```

Comment:
I have a small suggestion to move the picture up a little bit and place a border around it...

here is my suggested styling code:

   padding-bottom: 1rem;                   _(this i changed from 2.5rem)_
}
.image {
    margin: 0 0 40px 0;                         _(added this line)_
    border: solid 3px rgb(107, 3, 3);      _(added this line)_
    border-radius: 3px;                         _(added this line)_
    max-height: 40vh;                           _(this i changed from 45vh)_

Now....
![image](https://user-images.githubusercontent.com/89969770/168425622-b653c7d4-9f9d-40e3-87b2-96da5280bc4b.png)

My suggestion...
![image](https://user-images.githubusercontent.com/89969770/168425598-9ec087c1-61bb-406a-a342-fcb281574599.png)

Reply:
Good suggestions Mats, it makes the poster pop a bit more from the background 👍  I pushed the changes now